appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.appengine.demo.jettymain;
+
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.webapp.Configuration.ClassList;
+import org.eclipse.jetty.webapp.WebAppContext;
+
+/** Simple Jetty Main that can execute a WAR file when passed as an argument. */
+public class Main {
+
+  public static void main(String[] args) throws Exception {
+    if (args.length != 1) {
+      System.err.println("Usage: need a relative path to the war file to execute");
+      System.exit(1);
+    }
+    System.setProperty(
+      "org.eclipse.jetty.util.log.class",
+      "org.eclipse.jetty.util.log.StrErrLog");
+    System.setProperty("org.eclipse.jetty.LEVEL", "INFO");
+
+    // Create a basic Jetty server object that will listen on port 8080.
+    // Note: If you set this to port 0, a randomly available port will be
+    // assigned. You can find the assigned port in the logs or programmatically
+    // obtain it.
+    Server server = new Server(8080);
+
+    // The WebAppContext is the interface to provide configuration for a web
+    // application. In this example, the context path is being set to "/" so
+    // it is suitable for serving root context requests.
+    WebAppContext webapp = new WebAppContext();
+    webapp.setContextPath("/");
+    webapp.setWar(args[0]);
+    ClassList classlist = ClassList.setServerDefault(server);
+
+    // Enable Annotation Scanning.
+    classlist.addBefore(
```

Comment:
Useful, but should you also run the jetty quickstart in your pom?  https://www.eclipse.org/jetty/documentation/9.4.x/quickstart-webapp.html

Reply:
I see the benefits of using the jetty quickstart, but I'm not sure how to use it.